wp-content/themes/haruoshi_origin/functions.php

```diff
@@ -57,6 +57,14 @@ function enqueue_style() {
 			filemtime( get_stylesheet_directory() . '/dist/css/archive-works.css' ),
 		);
 	endif;
+	if ( get_post_type() === 'works' ) :
+		wp_enqueue_style(
+			'works',
```

Comment:
ここのハンドル名が一覧ページのCSSと被ってしまっているので
```suggestion
			'single-works',
```
とかがいいと思います。
それに伴い、archive-works.cssの読み込みの方のハンドル名も修正しましょうか。

Reply:
done https://github.com/oooshima/origin_wordpress/pull/28/commits/aba196be18770be61a69cd96b3a7335d3e981090